packages/tc-ui/src/pages/edit/template.jsx

```diff
@@ -138,6 +137,14 @@ const EditForm = props => {
 				</div>
 			</form>
 			<script src="https://cloud.tinymce.com/stable/tinymce.js" defer />
+			<script
+				type="application/json"
+				data-json="schema-data"
+				dangerouslySetInnerHTML={{
+					__html: JSON.stringify({ ...rawData.getAll() }),
+				}}
+			/>
+			<script src="https://cdnjs.cloudflare.com/ajax/libs/autolinker/3.11.1/Autolinker.min.js" />
```

Comment:
ah, now I see. If we can, it would be good to understand why autolinker cannot be bundled by webpack successfully